src/graphql/FlowLabelLimit/flow-label-limit.gql

```diff
@@ -0,0 +1,7 @@
+query FlowLabelLimit {
+  flow_concurrency_limit {
+    id
+    limit
+    name
```

Comment:
Note to reviewer: the confusion between `label` and `name` is a consequence of how this is implemented in the backend.